templates/install_puppet.ps1.erb

```diff
@@ -0,0 +1,262 @@
+# install_puppet.ps1
+<#

```

Comment:
On a similar note about this being a template... these inline docs won't inform the PowerShell runtime during test / debugging since it's not a PS1 until rendered.